app/models/miq_server/worker_management/monitor/quiesce.rb

```diff
@@ -3,9 +3,6 @@ module MiqServer::WorkerManagement::Monitor::Quiesce
 
   def workers_quiesced?
     # do a subset of the monitor_workers loop to allow for graceful exit
-    my_server.heartbeat
```

Comment:
I moved this up to the caller right after the `my_server.reload` so that I could do `check_not_responding; super`

Also https://github.com/ManageIQ/manageiq/pull/21679/files#diff-2760005b811135dbfa4f3b4c07682478cd96f0d0d5a506b0ba123e8923fcbc2aR45 is the only caller of this method so no need for other updates